Userland/Libraries/LibLine/Editor.cpp

```diff
@@ -2006,6 +2006,40 @@ VTState actual_rendered_string_length_step(StringMetrics& metrics, size_t index,
     return state;
 }
 
+static Result<char, Editor::Error> try_read_char_for_vt_dsr_with_retries()
```

Comment:
This feels very C
- random variables declared at the top of the function
- ...and not initialised